app/src/main/AndroidManifest.xml

```diff
@@ -22,6 +22,15 @@
         android:label="@string/APP_NAME"
         android:largeHeap="true"
         android:theme="@style/Hedvig.Theme">
+        <activity android:name=".feature.onbarding.ui.MoreOptionsActivity"></activity>
```

Comment:
```suggestion
        <activity android:name=".feature.onbarding.ui.MoreOptionsActivity" />
```

Additionally, we might want some of the other attributes we usually specify on the other activities. Maybe.